docs/pages/system/visually-hidden.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+
+const pageFilename = 'system/visually-hidden';
+const requireDemo = require.context('docs/src/pages/system/visually-hidden', false, /\.(js|tsx)$/);
+const requireRaw = require.context(
+  '!raw-loader!../../src/pages/system/visually-hidden',
+  false,
+  /\.(js|md|tsx)$/,
+);
+
+export default function Page({ demos, docs }) {
```

Comment:
I wonder about the name of the section, I wonder if we can't make it a bit broader:

- Tailwind puts it under an a11y section: https://tailwindcss.com/docs/screen-readers/#app
- Bootstrap puts it under a screen reader section: https://getbootstrap.com/docs/4.5/utilities/screen-readers/

Reply:
Was thinking about the same thing. Let's create Screen readers page and add this as a section? What do you think?

Reply:
Sounds great 👍